### PR TITLE
Update image repo name to crossplanecontrib

### LINKED ISSUE #142: Update image repo in installation example
### What problem are you facing?

Hey guys, amazing work you've been doing here.
I see that #112 changed the image repo, so from version 0.4.0 onwards a change is needed in the installation example:

`image: crossplane/provider-terraform-controller:v0.5.0` -> `image: crossplanecontrib/provider-terraform-controller:v0.5.0`
`package: crossplane/provider-terraform:v0.5.0` -> `package: v/provider-terraform:v0.5.0`

### How could Crossplane help solve your problem?

n/a

### PR DESCRIPTION
**Pull request by ltdeoliveira**
### Description of your changes

Update image repo name to crossplanecontrib on installation example.
Fixes #142 

I have:

- [ ] Read and followed Crossplane's [contribution process].
- [ ] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

Applied the example on a personal cluster and successfully installed version v0.5.0.

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

**Review by ytsarev — COMMENTED** [review]
Thanks a lot for the catch! Could you please comply with DCO just by making `git commit --amend -s` on your commit? :)

**Comment by ltdeoliveira** [comment]
> Thanks a lot for the catch! Could you please comply with DCO just by making `git commit --amend -s` on your commit? :)

@ytsarev sorry for the delay, life got in the way. Sure, once this is merged here I can also open the PR in the Upbound repo.

**Comment by ytsarev** [comment]
> Also, quick question, should I open new issues on the Upbound repo?

Yes, please, we are trying to consolidate in the official Upbound repo, meanwhile, the government of the project stays the same. 

Thanks a lot for your contribution!

File: examples/install.yaml [hunk 1]
```diff
@@ -6,14 +6,14 @@ metadata:
   labels:
     app: crossplane-provider-terraform
 spec:
-  image: crossplane/provider-terraform-controller:v0.2.0
+  image: crossplanecontrib/provider-terraform-controller:v0.5.0
```
Comment (ytsarev) [inline examples/install.yaml:9]: Actually this line can be removed after we switched to new 'fat' combined image where we have both controller and CRDs

**Review by ytsarev — CHANGES_REQUESTED** [review]
small nit to simplify install manifest

File: examples/install.yaml (hunk 1 above)
Comment (ltdeoliveira) [reply examples/install.yaml:9]: Nice, should I keep the `args: ["-d"]`?

Comment (ytsarev) [reply examples/install.yaml:9]: yes, so it will still keep the verbose logging

File: examples/install.yaml [hunk 2]
```diff
@@ -6,14 +6,13 @@ metadata:
...
-  image: crossplane/provider-terraform-controller:v0.2.0
   args: ["-d"]
 ---
 apiVersion: pkg.crossplane.io/v1
 kind: Provider
 metadata:
   name: crossplane-provider-terraform
 spec:
-  package: crossplane/provider-terraform:v0.2.0
+  package: crossplanecontrib/provider-terraform:v0.5.0
```
Comment (ytsarev) [inline examples/install.yaml:16]:
```suggestion
  package: crossplanecontrib/provider-terraform:v0.6.0
```
Sorry, last nit, let's include the latest release here

**Review by ytsarev — CHANGES_REQUESTED** [review]

**Review by ytsarev — APPROVED** [review]
@ltdeoliveira thank you!

**Pull request merged**